js/index.js

```diff
@@ -0,0 +1,69 @@
+const books = [{}];
```

Comment:
- [ ] there is no need to add an empty object inside this array, now there will always be an empty object in your array.

Reply:
Done as advised